Lib/test/test_call.py

```diff
@@ -760,6 +760,32 @@ def __call__(self, *args):
                 self.assertEqual(expected, meth(*args1, **kwargs))
                 self.assertEqual(expected, wrapped(*args, **kwargs))
 
+    def test_setvectorcall(self):
+        from _testcapi import function_setvectorcall
+        def f(num): return num + 1
+        num = 10
+        self.assertEqual(11, f(num))
+        function_setvectorcall(f)
+        # make sure specializer is triggered by running > 50 times
+        for _ in range(51):
+            self.assertEqual("overridden", f(num))
+
+    def test_setvectorcall_load_attr_specialization(self):
+        from _testcapi import function_setvectorcall
+
+        class X:
+            def __getattribute__(self, attr):
+                return attr
+
+        x = X()
+        # trigger LOAD_ATTR_GETATTRIBUTE_OVERRIDDEN specialization
+        for _ in range(8):
+            self.assertEqual("a", x.a)
```

Comment:
Please pull both `self.assertEqual` out of the loops. This itself is a `LOAD_ATTR` and there have been situations in the past where it confounded some of the test results when the test fails.

so
```python
assertEqual = self.assertEqual
...
```

Reply:
done